CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ allow LBs to revert to the old health check behaviour. The annotation and old he
   needs to be manually updated when using Let's Encrypt certificates. This is due to the certificate ID updating each time the
   certificate is rotated. If both annotations are provided, then `service.beta.kubernetes.io/do-loadbalancer-certificate-id` takes
   precedence.
+* Adding support for specifiying `loadBalancerSourceRanges` in the service spec. Source ranges take precedence over annotation based allow rules (`service.beta.kubernetes.io/do-loadbalancer-allow-rules`).
```

Comment:
The change log items right above have been released already as v0.1.51. I'm rather confused how the new entry added here does not lead to a conflict from [the current master change log](https://github.com/digitalocean/digitalocean-cloud-controller-manager/blob/0d3127f065dc0d25b88f3836c5029c2dfbc03507/CHANGELOG.md). 🤔 

The source range announcement should be the only unreleased change we have by now.

Reply:
I had an older state of master that I branched, that's why in mine this was still unreleased.
There's no conflict because git doesn't care about the headline movement and the line itself doesn't conflict.
I'll rebase on the current master and will move this one around.